rollbar/resource_user.go

```diff
@@ -97,6 +97,9 @@ func resourceUserCreate(ctx context.Context, d *schema.ResourceData, meta interf
 // specified.
 func resourceUserCreateOrUpdate(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
 	c := meta.(map[string]*client.RollbarAPIClient)[schemaKeyToken]
+
+	client.Mutex.Lock()
```

Comment:
Why are we deferring the unlock here but not explicitly lock/unlocking like we do in the rest of the PR?

Reply:
yeah, this one had more  API calls in between, I changed it